opendrift_leeway_webgui/leeway/templates/leeway/leewaysimulation_documentation.html

```diff
@@ -39,4 +93,65 @@ <h3>Interpreting Simulation Result</h3>
+              overflow:hidden;padding:10px 5px;word-break:normal;}
+            .tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+              font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+            .tg .tg-73oq{border-color:#000000;text-align:left;vertical-align:top}
+            .tg .tg-0lax{text-align:left;vertical-align:top}
+            </style>
+            <table class="tg">
+            <thead>
+              <tr>
+                <th class="tg-73oq">Latitude</th>
+                <th class="tg-73oq"><span style="font-weight:400;font-style:normal">37° 30'</span></th>
+                <th class="tg-0lax">-&gt; Last known Latitude</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td class="tg-73oq">Longitude</td>
+                <td class="tg-73oq"><span style="font-weight:400;font-style:normal">18°37'</span></td>
+                <td class="tg-0lax"><span style="font-weight:400;font-style:normal">-&gt; Last known Longitude</span></td>
+              </tr>
+              <tr>
+                <td class="tg-73oq">Object type</td>
+                <td class="tg-73oq"><span style="font-weight:400;font-style:normal">Fishing vessel, general (mean values)</span></td>
+                <td class="tg-0lax">-&gt; It was reported that it is a fishing vessel, but we do not know the exact type of fishing vessel so we choose general</td>
+              </tr>
+              <tr>
+                <td class="tg-73oq">Start time</td>
+                <td class="tg-73oq">2024-01-04 11:34:00</td>
+                <td class="tg-0lax">-&gt; The last position was reported at 12:34 local time, that is 11:34 in UTC</td>
+              </tr>
+              <tr>
+                <td class="tg-73oq">Duration</td>
+                <td class="tg-73oq">4</td>
+                <td class="tg-0lax">-&gt; You calculated that it takes you 4 hours to reach the last known position so you use that as the duration of the simulation. You could also choose a bit more but the result becomes harder to interpret.</td>
+              </tr>
+              <tr>
+                <td class="tg-73oq">Radius</td>
+                <td class="tg-73oq">1000</td>
+                <td class="tg-0lax">-&gt; The position was reported by an airplane and only precise to the degrees minutes so you keep the default radius of 1000m. </td>
+              </tr>
+            </tbody>
+            </table>
+    </p>
+    <p>
+        You recive an Email with the simulation result. It is clear the the boat is likely to drift North to South-East-East so you set your course to 171 to position 37.5°N 18,68°. From that point you should be able to spot the drifting vessel if the weather conditions are good.
+        <img src="555af619-11b3-462e-b560-78764290d2f7.png" alt="Simulation result example">
```

Comment:
That will not work. A bad but possible work around is to base64 encode the image and directly insert it into the template. Alternatively, we need to implement https://docs.djangoproject.com/en/5.0/howto/static-files/.